model.go

```diff
@@ -15,10 +15,22 @@ type FlagGroupModel struct {
 func (f *FlagGroupModel) FlagSummary() string {
 	out := []string{}
 	count := 0
+
 	for _, flag := range f.Flags {
-		if flag.Name != "help" {
+
+		ignoreInCount := map[string]bool{
```

Comment:
Would you mind making this a global please? Other than that, looks good to go, thanks!